MiscVBAFunctions/Modules/EnsureDictIUtil.bas

```diff
@@ -0,0 +1,44 @@
+Attribute VB_Name = "EnsureDictIUtil"
+Option Explicit
+
+Function EnsureDictI(Container As Variant) As Object
```

Comment:
@BartJan142 remember to add a unit test for this one. I think it would be to create a `dict()`, and then ensure you can't find a key with a different case, then transform it with `EnsureDictI()` and then do an assertion that you can find a key with a different casing.